src/ripple/app/tests/Flow_test.cpp

```diff
@@ -977,7 +977,7 @@ struct Flow_test : public beast::unit_test::suite
 
         auto const timeDelta = Env{*this}.closed ()->info ().closeTimeResolution;
 
-        for(auto const& t :{dcSoTime(), numAccSoTime()}){
+        for(auto const& t :{amendmentRIPD1141SoTime(), amendmentRIPD1141SoTime()}){
```

Comment:
I'm probably missing something, but doesn't this outer `for` loop set `t` to the same value twice?  Do we still need the outer `for`?